pages/services/spark/2.8.0-2.4.0/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: layout.pug
+title: Spark 2.8.0-2.4.0
+navigationTitle: Spark 2.8.0-2.4.0
+menuWeight: 1
```

Comment:
Can you update the two previous versions to be `2` and `-1`? This will move them down the actual docs site so that you'll only see the two latest versions displayed: https://docs.mesosphere.com/services/spark/

For `2.5.0-2.2.1` you can also set to `-1` so that we don't publicly support it any longer

Reply:
updated